pages/update-profile.js

```diff
@@ -0,0 +1,102 @@
+import { useUser } from '@/utils/context/user'
+import { useState, useEffect } from 'react'
+import { useRouter } from 'next/router'
+import { supabase } from '@/utils/supabase'
+
+const trueFalse = [
+  { id: 1, text: 'True', value: true },
+  { id: 2, text: 'False', value: false }
+]
+
+export default function UpdateProfile() {
+  const { activeUser } = useUser()
+  const [updateData, setUpdateData] = useState({
+    avatar: '',
+    location: '',
+    isPasswordProtected: '',
+    pagePassword: '',
+    sites: ''
+  })
+
+  useEffect(() => {
+    if (activeUser) {
+      setUpdateData({
+        avatar: activeUser.avatar,
+        location: activeUser.location,
+        isPasswordProtected: activeUser.isPasswordProtected,
+        pagePassword: activeUser.pagePassword,
+        sites: activeUser.sites
+      })
+    }
+  }, [activeUser])
+
+  const handleChange = (e) => {
+    e.preventDefault()
+    setUpdateData({ ...updateData, [e.target.id]: e.target.value })
+  }
+
+  const updateProfile = async (e) => {
+    e.preventDefault()
+    const { data, error } = await supabase
+      .from('profiles')
+      .update(updateData)
+      .eq('id', activeUser.id)
+  }
+  return activeUser ? (
```

Comment:
Same thing as the `create-release` template. Render the template, then above it, add your condition with whatever should be returned. I think it should be a signup/login form. Give the user an action to take, very good, much UX